src/components/Image/Image.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+import styles from './Image.module.scss';
+
+import { addQueryParams } from '#src/utils/formatting';
+
+type Props = {
+  className?: string;
+  src?: string;
+  fallbackSrc?: string;
```

Comment:
Suggest using the `ImageData` type here so you don't manually have to assign both props for each usages